### PR TITLE
CHAOS-1660: new topic HCE/SRM integration

### PR DESCRIPTION
**Pull request by StrawberryPad**
[**Link to preview**](https://64623d3562fbce203ecd43c6--harness-developer.netlify.app/docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm)

Early access

### CODE REVIEW

File: docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: Use chaos experiments with SRM
+sidebar_position: 35
+---
+
+Within Harness, you can see the impact of chaos faults on your target environment by using [Service Reliability Management (SRM)](/docs/service-reliability-management).
+To do this, you connect any chaos fault within an experiment to monitored services. A monitored service in Harness is a combination of a service and environment that [SRM](/docs/service-reliability-management) monitors for changes, events, and health trends. 
+
+For example, let's say you use SRM to monitor services and to correlate change events to service health. Now you want to see how one or more chaos faults impacts your target environment. By adding the target environment as one or more monitored services, and linking faults to those services, you can see the impacts on service health in SRM when you run experiments containing those faults.
```
Comment (amityt) [inline docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md:9]: Can we rephrase this to:
Now you want to see how one or more chaos faults impact your target environment -> Now you want to see how one or more chaos faults impact the services in the target environment

File: docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+* [Create a monitored service](/docs/service-reliability-management/monitored-service/create-monitored-service).
+* [Create your first SLO](/docs/service-reliability-management/getting-started/create-first-slo)
+
+## Connect a fault to a monitored service
+
+You can connect any fault within a chaos experiment to one or more monitored services.
+
+To connect a fault to a monitored service:
+
+1. In Harness, select **Chaos > Chaos Experiments**, and then find and select an experiment.
```
Comment (amityt) [inline docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md:26]: find and select an experiment -> create an experiment

File: docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	
+	![The SRM SLOs dashboard, listing 3 SLOs, and their associated monitored services](./static/srm/srm-slo-dashboard.png) 
+
+	Here you can find the monitored service(s) corresponding to your target environment.
+
+1. Use the filters at the top to narrow down the list of SLOs.
+
+	![SLOs dashboard filters](./static/srm/srm-slo-filters.png)
+
+1. Go to **Chaos > Changes** to get another view of the data.
```
Comment (amityt) [inline docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md:55]: **Service Reliability > Changes**

**Review by amityt — CHANGES_REQUESTED** [review]
The changes look good to me @StrawberryPad ! 🚀 
Requested some minor changes 😅

File: docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md (hunk 2 above)
Comment (StrawberryPad) [reply docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md:26]: @amityt Just checking: you can connect a fault to services within an existing experiment, OR create an experiment and connect a fault to services, right? (the above assumed you're going to add one to an existing experiment)

Comment (amityt) [reply docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md]: Yes, we can add services to existing experiments as well 👍🏼

**Review by amityt — DISMISSED** [review]
LGTM 🚀

File: docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+1. Below the timeline, in the Changes section, select a change event to see [SLO details](/docs/service-reliability-management/getting-started/slo-dashboard#slo-details-page).
+
+	Here you can see performance trends and error budget for the SLO, among other details. 
+
+1. To go to the chaos experiment corresponding to this change event, select **View Chaos Experiment**.
+
+## More information
+
+For more information on Harness SRM and monitored services, go to these topics to:
```
Comment (SudheendraKatte) [inline docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md:77]:
```suggestion
For more information on Harness SRM and monitored services, go to these topics:
```

File: docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+---
+
+:::note
+This feature is currently behind two feature flags: `CHAOS_SRM_EVENT` and `SRM_INTERNAL_CHANGE_SOURCE_CE`. Contact Harness Support to enable the feature.
+:::
+
+Within Harness, you can see the impact of chaos faults on your target environment by using [Service Reliability Management (SRM)](/docs/service-reliability-management).
+To do this, you connect any chaos fault within an experiment to monitored services. A monitored service in Harness is a combination of a service and environment that [SRM](/docs/service-reliability-management) monitors for changes, events, and health trends. 
+
+For example, let's say you use SRM to monitor services and to correlate change events to service health. Now you want to see how one or more chaos faults impacts services in your target environment. By adding the target environment as one or more monitored services, and linking faults to those services, you can see the impacts on service health in SRM when you run experiments containing those faults.
```
[the remark this answers is not in the document]
Comment (StrawberryPad) [reply docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md:13]: Thanks, Sudhi. The original is correct as it relates to multiple services in the target env. :-)

File: docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+1. Use the filters at the top to narrow down the list of SLOs.
+
+	![SLOs dashboard filters](./static/srm/srm-slo-filters.png)
+
+1. Go to **Service Reliability > Changes** to get another view of the data.
+
+	![SRM **Changes** dashboard](./static/srm/srm-changes-dashboard.png)
+
+	The top part of the [Changes dashboard](/docs/service-reliability-management/change-impact-analysis/change-impact-analysis-changes-dash-board) displays a Changes Timeline, including a separate line for **Chaos Experiments**. Chaos events along this timeline are indicated by diamonds icons.
```
[the remark this answers is not in the document]
Comment (StrawberryPad) [reply docs/chaos-engineering/configure-chaos-experiments/experiments/use-chaos-with-srm.md:63]: nicely caught! should be "diamond"
